rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/user/UserController.java

```diff
@@ -207,4 +211,59 @@ public RepositoryLinksResource process(RepositoryLinksResource resource) {
     private HalResource<User> createHalUser(User sw360User) {
         return new HalResource<>(sw360User);
     }
+
+    @Operation(
+            summary = "List all of the service's users.",
+            description = "List all of the service's users.",
+            tags = {"Users"}
+    )
+    @GetMapping(value = USERS_URL + "/name")
+    public ResponseEntity<CollectionModel<EntityModel<User>>> getUsersByName(
```

Comment:
Instead of creating a new endpoint. Kindly add a parameter in the existing get user endpoint viz. `/api/users?name=xxx`.